src/axom/lumberjack/tests/lumberjack_serial_main.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "gtest/gtest.h"
```

Comment:
Ok, so the main change to the test source would be that we'd need to add some extra files to each component with a `main`, and remove the existing `main`s.  Seems like a reasonable tradeoff.

Another benefit would be the reduced number of test targets.

Reply:
I purposefully left the test targets the same. But yes we could reduce this further as well.

like lumberjack_serial, lumberjack_mpi,.. etc